frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -65,7 +75,11 @@ export const ClickableDateContainer = (props: { date: Date; eventPresent?: boole
 
   return (
     <DateContainer
-      onClick={() => DateContainerClickHandler(assignedDateMonth)}
+      onClick={
+        parseInt(params.isEndDate) === 0
+          ? () => DateContainerClickHandler(assignedDateMonth)
+          : () => DateContainerEndDateHandler(props.date.getTime() / 1000)
+      }
```

Comment:
I think you should discuss with @marcus-ong-qy regarding your implementation of this part of the component as the logic seems to be different!